docs/performanceprofile/performance_addon.md

```diff
@@ -37,72 +37,74 @@ The deployment will be retried in a loop until everything is deployed successful
 In CI the `test/e2e/performanceprofile/cluster-setup/ci-cluster/performance/` dir will be used. The difference is that the CI cluster will deploy
 the PerformanceProfile in the test code, while the `manual` cluster includes it in the kustomize based deployment.
 
-
 Now you need to label the nodes which should be tuned. This can be done with
 
-```
+```cli
```

Comment:
Why cli here vs. shell in the two other changes, is that intentional?

Reply:
Nope, just trying to see which one renders better. 
I will change it to `shell` as there is no difference .

Reply:
done